pkg/api/vm/formatter.go

```diff
@@ -1,25 +1,93 @@
 package vm
 
 import (
+	"strings"
+
 	"github.com/rancher/apiserver/pkg/types"
+	ctlkubevirtv1alpha3 "github.com/rancher/harvester/pkg/generated/controllers/kubevirt.io/v1alpha3"
 	"github.com/rancher/steve/pkg/resources/common"
+
+	v1alpha3 "kubevirt.io/client-go/api/v1alpha3"
+	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
 )
 
 const (
-	startVM   = "startVM"
-	stopVM    = "stopVM"
-	restartVM = "restartVM"
+	startVM      = "startVM"
+	stopVM       = "stopVM"
+	restartVM    = "restartVM"
+	unmountCDRom = "unmountCDRom"
```

Comment:
Maybe `ejectCdrom` is a better name for the action? it sounds more common to me and we have the `eject` Linux command.